Automattic/wp-calypso
# ThemeCollection: Add a show more link that points the user to a themes filter

Related to https://github.com/Automattic/dotcom-forge/issues/3959

## Proposed Changes

* Add a new showMoreLink prop to the ThemeCollection component

## Testing Instructions
- Use the Calypso live link and go to /themes in incognito
- Enable the feature flag: flags=themes/discovery-lots
- Notice an unstyled Show more link on the page
- Click on the Premium one and on the Partner one
- Notice that you are redirected to /themes/premium and /themes/marketplace


*

## Pre-merge Checklist
